.gitignore

```diff
@@ -19,8 +19,6 @@
 
 node_modules
 
-/node_package/lib
```

Comment:
Why are you removing this?

Reply:
These are the source files: https://github.com/shakacode/react_on_rails/tree/master/node_package/src

lib is generated.

Reply:
So I can test the branch without getting you to make a new release.